styleguide/components/pagination/pagination.stories.js

```diff
@@ -1,5 +1,5 @@
 import docs from './pagination-docs.mdx';
-import templateExample from './_example.html';
+import templateExample from '../../examples/pagination/with_fifth_page.html';
```

Comment:
What happens if this page hasn't been generated yet? Are we going to force a run of the new rake task during the main build?

If the answer to this is "the files are committed to git", how are we going to ensure people remember to regenerate them if they change the component.

Reply:
Running the task during the main build would be a bit too late, so largely relying on the files being committed to git. But what we likely want to do as a future iteration is have a watcher that runs alongside the storybook dev server which regenerates them then. But open to other suggestions. This is kinda what we do now with the current webpack loader approach, although it's a bit flaky.

Reply:
This sounds like a good approach.

Reply:
👍  Ended up adding a watcher in this PR, have updated the description to cover that

Reply:
Can we have the rake task in the CI process so we know the docs will always be correct on release, regardless of whether someone has remembered to run the task / edited haml files with the watcher active?